docs/getting-started/ios/index.md

```diff
@@ -3,3 +3,56 @@
 
 This page describes how to get started building an application with Warp components.
 
+## 1. Integrate Warp
+
+A guide on how to integrate Warp into your project.
+
+### Swift Package Manager
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/warp-ds/warp-ios.git")
+]
+```
+
+## 2. Use Warp components
+
+Creating component:
+
+    Button:
+
+        Warp.Button(type: <#T##Warp.ButtonType#>, title: <#T##String#>, imageSystemName: <#T##String?#>, action: <#T##() -> Void#>, size: <#T##Warp.ButtonSize#>, isEnabled: <#T##Bool#>, fullWidth: <#T##Bool#>, colorProvider: <#T##ColorProvider#>)
+
+        Warp.Button.create(for: .primary, title: "Warp", icon: Image(systemName: "plus")) {
+            print("Warp button")
+       }
```

Comment:
Can you wrap all the code examples with code formatting, i .e. like you are doing here: 
<img width="636" alt="image" src="https://github.com/warp-ds/tech-docs/assets/5851332/84afc721-4577-4dc3-878d-fd1fc13a9c43">

Reply:
Or wait... I think the examples of how to use the button component belongs in the "component" section, i.e. [here](https://warp-ds.github.io/tech-docs/components/buttons/) for button. If you scroll down you can choose between web frameworks (React, Vue, Elements) and Android. We want to add iOS here as well. But that can be done in a separate task.